### ISSUE TITLE
Cannot resolve symbol 'UseWebApiHelpPage'

### ISSUE THREAD

**Issue by NZTSL** [DalSoft/DalSoft.WebApi.HelpPage#1]
Hi,

I added the package via Nuget but then got the **Cannot resolve symbol 'UseWebApiHelpPage'** error when adding this line in my code:

``` csharp
app.UseWebApiHelpPage(config);
```

Any idea ?

Thanks.

**Comment by DalSoft** [comment]
Are you missing? 

``` cs
using  DalSoft.WebApi.HelpPage;
```

**Comment by NZTSL** [comment]
Seems I'm missing the dll actually.

**Comment by NZTSL** [comment]
Ok the dll was in packages folder but I had to reference it manually.

I have another error now:

> The type or namespace name 'DalSoft' could not be found (are you missing a using directive or an assembly reference?)

on this line:

``` csharp
using  DalSoft.WebApi.HelpPage;
```

**Comment by NZTSL** [comment]
Is it compatible with .NET 4.5 or 4.5.1 only ?

**Comment by NZTSL** [comment]
Ok upgraded the project to 4.5.2, works fine now, thanks.

**Comment by DalSoft** [comment]
Hi sorry just realized what has happened, the project has been build targeting .Net 4.5.2 which is why NuGet skips referencing the dll. I fix it to tonight 4.5 as it doesn't need to be .Net 4.5.2.

**Comment by NZTSL** [comment]
Ok thanks!
